src/Illuminate/Support/Collection.php

```diff
@@ -617,6 +617,17 @@ public function whereInStrict($key, $values)
         return $this->whereIn($key, $values, true);
     }
 
+    /**
+     * Filter items where the given key between values.
+     * @param $key
+     * @param $values
+     * @return Collection
```

Comment:
Please use FQN and add proper types and spacing to the params. Add a white line above the params.

Reply:
Done